app/services/editions/publishing/EditionsPublishing.scala

```diff
@@ -17,24 +17,36 @@ class EditionsPublishing(publishedBucket: EditionsBucket, previewBucket: Edition
     previewBucket.putIssue(previewIssue)
   }
 
-  def publish(issue: EditionsIssue, user: User, now: OffsetDateTime) = {
+  def proof(issue: EditionsIssue, user: User, now: OffsetDateTime) = {
+
+    val action = PublishAction.proof
+
     // Bump the recently published counters
     val versionId = db.createIssueVersion(issue.id, user, now)
 
     val markers = Markers.appendEntries(
-      Map (
+      Map(
+        "issue-action" -> action.toString,
         "issue-id" -> issue.id,
         "issue-date" -> issue.issueDate.toString,
         "version" -> versionId,
         "user" -> user.email
       ).asJava
     )
 
-    Logger.info(s"Publishing issue ${issue.id}")(markers)
+    Logger.info(s"Uploading $action request for issue ${issue.id} to S3")(markers)
+
+    val publishedIssue = issue.toPublishableIssue(versionId, action)
+
+    // Archive a copy
+    publishedBucket.putIssue(publishedIssue)
```

Comment:
Does proofing and publishing share the same bucket?

Reply:
Yes, for notification purposes.  This keeps the listener logic simpler.